jekyll/_includes/partials/server/latest/installation/phase-3.adoc

```diff
@@ -288,10 +289,30 @@ endif::env-aws[]
 [#nomad-servers]
 == 2. Nomad servers
 
-Now that you have successfully deployed your Nomad clients and have the permission resources, you can configure the Nomad Servers.
+In the previous section you deployed your Nomad clients and have the IAM resources provisioned. Now you can deploy and configure the Nomad Servers.
+
+[#where-to-deploy-nomad-servers]
+=== a. Where to deploy nomad servers
+Nomad Servers are by default deployed within your CircleCI server cluster. However, Nomad Servers may be deployed externally. As with the Nomad clients, you can use the terraform module CircleCI provides to deploy your Nomad Servers or as a guide for how such a deployment might look.
+* If you wish to deploy your Nomad servers inside your CircleCI server cluster, continue to <<nomad-gossip-encryption-key>> below. 
+* If you wish to deploy your Nomad servers externally, follow these steps:
+
+.. Update your Terraform by setting the variable `nomad_server_enabled` to `true`.
+.. Run `terraform apply` to apply your changes. This will deploy Nomad Servers with the default configurations to the existing infrastructure you created when setting up the Nomad clients earlier. For a full list of the Nomad Server options, refer to the `variables.tf` in our link:https://github.com/CircleCI-Public/server-terraform/blob/main/nomad-aws/variables.tf[public repository].
```

Comment:
Should we have a GCP section, too?

Reply:
good catch. i'm hoping my update addresses it well enough without needing a whole new section